docs/containerignore.5.md

```diff
@@ -0,0 +1,87 @@
+% "containerignore" "28" "Sep 2021" "" "Container User Manuals"
```

Comment:
IDK, but can we rename this to `.containerignore` or does that throw man pages for a loop?

Reply:
I am not sure.  Users are not going to type that though. I did setup a link to this from .containerfile.